Java/hamcrest/src/test/java/deque/exampleTest.java

```diff
@@ -0,0 +1,73 @@
+package deque;
+
+import java.io.File;
+import java.io.IOException;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import com.deque.html.axecore.results.Results;
+import com.deque.html.axedevtools.matchers.*;
+import com.deque.html.axedevtools.selenium.*;
+import com.deque.html.axedevtools.selenium.reporter.*;
+
+import static com.deque.html.axedevtools.matchers.IsAuditedForAccessibility.isAuditedForAccessibility;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class exampleTest
+{
+    static String reporter = new File("src/test/resources/reporter-cli-macos").getAbsolutePath();
+    public static AxeReportingOptions _reportOptions = new AxeReportingOptions();
+    public static WebDriver driver = null;
+    public static AxeDriver axedriver = null;
+    public static AxeSelenium axeSelenium = null;
+    public static String Logger;
+    public static String Destination;
+
+    @BeforeClass
+    public static void initiate_drivers(){
+        _reportOptions = new AxeReportingOptions();
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
+        axedriver = new AxeDriver(driver);
+        axeSelenium = new AxeSelenium();
+        AxeConfiguration.configure()
+                .testSuiteName("Test")
+                .outputDirectory("axe-json-reports/");
+        Logger =  new File("axe-json-reports/").getAbsolutePath();
+        Destination =  new File("a11y-results/").getAbsolutePath();
+
+    }
```

Comment:
```suggestion
    private String reporter = new File("src/test/resources/reporter-cli-macos").getAbsolutePath();
    private AxeReportingOptions _reportOptions;
    private WebDriver driver;
    private AxeDriver axedriver;
    private AxeSelenium axeSelenium;
    private String Logger;
    private String Destination;

    @Before
    public void init() {
        _reportOptions = new AxeReportingOptions();
        WebDriverManager.chromedriver().setup();
        driver = new ChromeDriver();
        axedriver = new AxeDriver(driver);
        axeSelenium = new AxeSelenium();
        AxeConfiguration.configure()
                .testSuiteName("Test")
                .outputDirectory("axe-json-reports/");
        Logger =  new File("axe-json-reports/").getAbsolutePath();
        Destination =  new File("a11y-results/").getAbsolutePath();

    }
```

We don't need to use static variables here, we can use Junit's `@Before` which will run before each test. 

We could use JUnit 5 - which has `@BeforeAll` which will run only once (before all tests) and remove the use of `static`